scripts/daily_update.py

```
@@ -23,8 +23,134 @@
+                        )
+                        continue
+                    if result:
+                        stats["records_parsed"] += 1
+                        stats["records_imported"] += 1
+                    else:
+                        stats["records_failed"] += 1
+            finally:
+                try:
+                    jvlink.jv_close()
+                except Exception:
+                    pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)